kernel/src/tty/core_env.rs

```diff
@@ -35,6 +36,45 @@ pub fn init_core_env(env: &Rc<RefCell<Env>>) {
             env: env.clone(),
         },
     );
+    env.borrow_mut().set(
+        "read-string",
+        MalType::Builtin {
+            eval: read_string,
+            args: Box::new(MalType::List(vec![MalType::Symbol("a".to_string())])),
+            body: Box::new(MalType::Nil),
```

Comment:
nit: So I didn't see the code for entire lisp interpreter, but all builtins declared here seem to not have a body. Is this on purpose? Maybe a `constructor` that would initiate this to `MalType::Nil` would be nice?

Reply:
For historical reasons, Builtins and function were the same which is why it's here, you're right this can be safely removed for now.